### PR TITLE
New guide: Knot Authoritative DNS

### PR DESCRIPTION
**Pull request by neelchauhan**
#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by alemorvan** [comment]
Very good. Thx @neelchauhan

File: docs/guides/dns/knot_authoritative_dns.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+author: Neel Chauhan
+contributors:
+tested_with: 9.4
+tags:
+  - dns
+---
+
+# Knot Authoritative DNS
+
+An alternative to BIND, [Knot DNS](https://www.knot-dns.cz/) is an modern authoritative-only DNS server maintained by the Czech domain registry [CZ.NIC](https://www.nic.cz/).
```
Comment (SergeCroise) [inline docs/guides/dns/knot_authoritative_dns.md:12]:
```suggestion
An alternative to BIND, [Knot DNS](https://www.knot-dns.cz/) is a modern authoritative-only DNS server maintained by the Czech domain registry [CZ.NIC](https://www.nic.cz/).
```

File: docs/guides/dns/knot_authoritative_dns.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+Before making changes to any configuration file, move the original installed working file, `knot.conf`:
+
+```bash
+mv /etc/knot/knot.conf /etc/knot/knot.conf.orig
+```
+
+That will help in the future if the introduction of errors into the configuration file occurs. It is *always* a good idea to make a backup copy before making changes.
+
+Edit the *knot.conf* file. The author is using *vi* , but you can substitute your favorite command line editor:
```
Comment (SergeCroise) [inline docs/guides/dns/knot_authoritative_dns.md:48]:
```suggestion
Edit the *knot.conf* file. The author is using *vi*, but you can substitute your favorite command line editor:
```

**Review by SergeCroise — APPROVED** [review]

**Review by sspencerwire — APPROVED** [review]
Thanks @neelchauhan ... light editing to follow

**Pull request merged**